src/main/java/net/imglib2/mesh/Meshes.java

```diff
@@ -375,4 +376,58 @@ public static void scale( final Mesh mesh, final double[] scale )
 			vertices.set( i, x * scale[ 0 ], y * scale[ 1 ], z * scale[ 2 ] );
 		}
 	}
+
+	public static BufferMesh merge( final Iterable< Mesh > meshes )
+	{
+		int nVertices = 0;
+		int nTriangles = 0;
+		for ( final Mesh mesh : meshes )
+		{
+			nVertices += mesh.vertices().size();
+			nTriangles += mesh.triangles().size();
+		}
+
+		final BufferMesh out = new BufferMesh( nVertices, nTriangles );
+		final BufferMesh.Vertices vOut = out.vertices();
+		final BufferMesh.Triangles tOut = out.triangles();
+
+		for ( final Mesh mesh : meshes )
+		{
+			// Add vertices.
+			final Vertices vIn = mesh.vertices();
+			final int[] inToOutMap = new int[ ( int ) vIn.size() ];
```

Comment:
The cast to `int` here will be unnecessary after rebasing over top of the `size`/`sizel` change, right? But it would be nice if this routine supported larger meshes; is this big honkin' array really necessary here?